### PR TITLE
Pioneer 18 Leg 1

### PR DESCRIPTION
**Pull request by cdobs**
Updates deployment lines for gliders 380 and 388, and adds Pioneer 18 AUV deployments

### CODE REVIEW

**Comment by cdobs** [comment]
AUV config can be reviewed in the FCP: 
AUV deploy times/location can be reviewed in P18 QLR: 3204-01802

File: deployment/CP05MOAS-A6264_Deploy.csv [hunk 1]
```diff
@@ -110,4 +110,18 @@ AR61,,AR61,,CP05MOAS-A6264-02-DOSTAN000,16,1,2021-11-04T18:34:00,2021-11-05T20:1
...
+AR66,,AR66,,CP05MOAS-A6264-02-DOSTAN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-02909,40.21065,-71.08783,,,,abbreviated mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-03-CTDAVN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00527,40.21065,-71.08783,,,,abbreviated mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-04-NUTNRN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-00367,40.21065,-71.08783,,,,abbreviated mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-05-ADCPAN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,40.21065,-71.08783,,,,abbreviated mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-06-PARADN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50267,40.21065,-71.08783,,,,abbreviated mission due to time constraints (AL-1 to AL-2),,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-A6264_Deploy.csv:127]: looks good; maybe add 'along-shelf' in between abbreviated & mission for the second AUV deployment just for consistency.

File: deployment/CP05MOAS-GL380_Deploy.csv [hunk 2]
```diff
@@ -65,9 +65,9 @@ TS20211214,,TS20220221,,CP05MOAS-GL380-02-FLORTM000,11,1,2021-12-15T07:36:00,202
...
-AR66,,,,CP05MOAS-GL380-05-PARADM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50152,39.93425,-70.58905,,1000,1000,FZ-2,,
\ No newline at end of file
+AR66,,,,CP05MOAS-GL380-00-ENG000000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGCON-GLDRCP-00380,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL380-01-ADCPAM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-ADCPAM-50867,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL380-02-FLORTM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-FLORTM-04251,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL380-03-CTDGVM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-CTDGVM-09037,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL380-04-DOSTAM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-DOSTAM-00171,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,,,CP05MOAS-GL380-05-PARADM000,12,1,2022-04-10T16:20:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50152,39.93425,-70.58905,,1000,1000,FZ-1,,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL380_Deploy.csv:73]: good - moved to FZ-1 as noted in slack channel.

File: deployment/CP05MOAS-GL388_Deploy.csv [hunk 3]
```diff
@@ -77,9 +77,9 @@ AR66,,AR66,,CP05MOAS-GL388-02-FLORTM000,13,1,2022-04-10T16:00:00,2022-04-10T22:5
...
-AR66,,,,CP05MOAS-GL388-05-PARADM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50238,39.9338,-70.66867,,1000,1000,FZ-1,,
\ No newline at end of file
+AR66,,,,CP05MOAS-GL388-00-ENG000000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.9338,-70.66867,,1000,1000,SS-1,,
+AR66,,,,CP05MOAS-GL388-01-ADCPAM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-43106,39.9338,-70.66867,,1000,1000,SS-1,,
+AR66,,,,CP05MOAS-GL388-02-FLORTM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-04467,39.9338,-70.66867,,1000,1000,SS-1,,
+AR66,,,,CP05MOAS-GL388-03-CTDGVM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-09363,39.9338,-70.66867,,1000,1000,SS-1,,
+AR66,,,,CP05MOAS-GL388-04-DOSTAM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00418,39.9338,-70.66867,,1000,1000,SS-1,,
+AR66,,,,CP05MOAS-GL388-05-PARADM000,14,1,2022-04-11T16:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50238,39.9338,-70.66867,,1000,1000,SS-1,,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL388_Deploy.csv:85]: good - moved to SS-1 as noted in slack channel

**Review by rtravis-whoi — APPROVED** [review]
for AUV deploy csv - maybe add 'along-shelf' in between abbreviated & mission for the second AUV deployment just for consistency.

File: deployment/CP05MOAS-A6263_Deploy.csv [hunk 4]
```diff
@@ -69,13 +69,13 @@ WJ20190709,,WJ20190709,,CP05MOAS-A6263-03-CTDAVN000,10,1,2019-07-10T12:41:00,201
...
+AR39,,AR39,,CP05MOAS-A6263-03-CTDAVN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-CTDAVN-00424,40.205267,-71.0940167,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
+AR39,,AR39,,CP05MOAS-A6263-04-NUTNRN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-NUTNRN-00366,40.205267,-71.0940167,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
+AR39,,AR39,,CP05MOAS-A6263-05-ADCPAN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.205267,-71.0940167,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
+AR39,,AR39,,CP05MOAS-A6263-06-PARADN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50170,40.205267,-71.0940167,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-A6263_Deploy.csv:78]: updates to notes look good!

File: deployment/CP05MOAS-A6264_Deploy.csv [hunk 5]
```diff
@@ -110,4 +110,18 @@ AR61,,AR61,,CP05MOAS-A6264-02-DOSTAN000,16,1,2021-11-04T18:34:00,2021-11-05T20:1
...
+AR66,,AR66,,CP05MOAS-A6264-02-DOSTAN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-02909,40.21065,-71.08783,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-03-CTDAVN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00527,40.21065,-71.08783,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-04-NUTNRN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-00367,40.21065,-71.08783,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-05-ADCPAN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,40.21065,-71.08783,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
+AR66,,AR66,,CP05MOAS-A6264-06-PARADN000,18,1,2022-04-16T00:04:00,2022-04-16T10:46:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50267,40.21065,-71.08783,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-A6264_Deploy.csv:127]: updates to notes look good!

**Review by rtravis-whoi — APPROVED** [review]
updates to AUV notes look good!

File: deployment/CP05MOAS-A6263_Deploy.csv [hunk 6]
```diff
@@ -41,20 +41,20 @@ WJ20180423,,WJ20180423,,CP05MOAS-A6263-03-CTDAVN000,6,1,2018-04-24T13:05:00,2018
...
+AR34,,AR34,,CP05MOAS-A6263-02-DOSTAN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-02901,40.201167,-71.09885,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
+AR34,,AR34,,CP05MOAS-A6263-03-CTDAVN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-CTDAVN-00424,40.201167,-71.09885,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
+AR34,,AR34,,CP05MOAS-A6263-04-NUTNRN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-NUTNRN-00366,40.201167,-71.09885,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
+AR34,,AR34,,CP05MOAS-A6263-05-ADCPAN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.201167,-71.09885,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
+AR34,,AR34,,CP05MOAS-A6263-06-PARADN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50267,40.201167,-71.09885,,,,abbreviated along-shelf mission due to time constraints (AL-1 to AL-2),,
```
Comment (snwhite) [inline deployment/CP05MOAS-A6263_Deploy.csv:57]: Looks good.

File: deployment/CP05MOAS-A6263_Deploy.csv (hunk 4 above)
Comment (snwhite) [reply deployment/CP05MOAS-A6263_Deploy.csv:78]: Looks good.

File: deployment/CP05MOAS-A6264_Deploy.csv [hunk 7]
```diff
@@ -41,20 +41,20 @@ WJ20180423,,WJ20180423,,CP05MOAS-A6264-03-CTDAVN000,6,1,2018-04-24T06:40:00,2018
...
+WJ20190228,,WJ20190228,,CP05MOAS-A6264-05-ADCPAN000,8,1,2019-03-01T03:57:00,2019-03-01T21:53:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.92753,-70.93248,,,,abbreviated across-shelf mission due to time constraints (AC-1 to AC-2),,
+WJ20190228,,WJ20190228,,CP05MOAS-A6264-06-PARADN000,8,1,2019-03-01T03:57:00,2019-03-01T21:53:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50266,39.92753,-70.93248,,,,abbreviated across-shelf mission due to time constraints (AC-1 to AC-2),,
 AR34,,AR34,,CP05MOAS-A6264-00-ENG000000,9,1,2019-04-05T22:12:00,2019-04-07T02:14:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,39.92028,-70.9071,,,,across-shelf survey,,
 AR34,,AR34,,CP05MOAS-A6264-01-FLORTN000,9,1,2019-04-05T22:12:00,2019-04-07T02:14:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01646,39.92028,-70.9071,,,,across-shelf survey,,
 AR34,,AR34,,CP05MOAS-A6264-02-DOSTAN000,9,1,2019-04-05T22:12:00,2019-04-07T02:14:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-02909,39.92028,-70.9071,,,,across-shelf survey,,
```
Comment (snwhite) [inline deployment/CP05MOAS-A6264_Deploy.csv:60]: Looks good.

File: deployment/CP05MOAS-A6264_Deploy.csv [hunk 8]
```diff
@@ -69,13 +69,13 @@ AR34,,AR34,,CP05MOAS-A6264-03-CTDAVN000,10,1,2019-04-10T17:00:00,2019-04-11T17:0
...
+WJ20190617,,WJ20190617,,CP05MOAS-A6264-03-CTDAVN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00425,39.93057,-70.90167,,,,abbreviated across-shelf mission due to time constraints (AC-1 to AC-2),,
+WJ20190617,,WJ20190617,,CP05MOAS-A6264-04-NUTNRN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-00367,39.93057,-70.90167,,,,abbreviated across-shelf mission due to time constraints (AC-1 to AC-2),,
+WJ20190617,,WJ20190617,,CP05MOAS-A6264-05-ADCPAN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.93057,-70.90167,,,,abbreviated across-shelf mission due to time constraints (AC-1 to AC-2),,
+WJ20190617,,WJ20190617,,CP05MOAS-A6264-06-PARADN000,11,1,2019-06-18T16:14:00,2019-06-19T16:12:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50180,39.93057,-70.90167,,,,abbreviated across-shelf mission due to time constraints (AC-1 to AC-2),,
```
Comment (snwhite) [inline deployment/CP05MOAS-A6264_Deploy.csv:78]: Looks good.

File: deployment/CP05MOAS-A6264_Deploy.csv [hunk 9]
```diff
@@ -90,13 +90,13 @@ WJ20190709,,WJ20190709,,CP05MOAS-A6264-03-CTDAVN000,13,1,2019-07-11T13:50:00,201
...
+AR39,,AR39,,CP05MOAS-A6264-02-DOSTAN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01519,39.92068,-70.764267,,,,abbreviated across-shelf mission due to time constraints (AC-4 to AC-1),,
+AR39,,AR39,,CP05MOAS-A6264-03-CTDAVN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00425,39.92068,-70.764267,,,,abbreviated across-shelf mission due to time constraints (AC-4 to AC-1),,
+AR39,,AR39,,CP05MOAS-A6264-04-NUTNRN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-00367,39.92068,-70.764267,,,,abbreviated across-shelf mission due to time constraints (AC-4 to AC-1),,
+AR39,,AR39,,CP05MOAS-A6264-05-ADCPAN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.92068,-70.764267,,,,abbreviated across-shelf mission due to time constraints (AC-4 to AC-1),,
+AR39,,AR39,,CP05MOAS-A6264-06-PARADN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50180,39.92068,-70.764267,,,,abbreviated across-shelf mission due to time constraints (AC-4 to AC-1),,
```
Comment (snwhite) [inline deployment/CP05MOAS-A6264_Deploy.csv:99]: Looks good.

File: deployment/CP05MOAS-A6264_Deploy.csv (hunk 5 above)
Comment (snwhite) [reply deployment/CP05MOAS-A6264_Deploy.csv:127]: Looks good.

File: deployment/CP05MOAS-GL380_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL380_Deploy.csv:73]: Looks good.

File: deployment/CP05MOAS-GL388_Deploy.csv (hunk 3 above)
Comment (snwhite) [reply deployment/CP05MOAS-GL388_Deploy.csv:85]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**